lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -2056,6 +2184,7 @@ def main():
         guest_id=dict(type='str'),
         disk=dict(type='list', default=[]),
         cdrom=dict(type='dict', default={}),
+        floppy=dict(type='dict', default={}),
```

Comment:
I think we should start using sub options. Part of it will need to come in a sort of cleanup PR, but we can start of properly with newly added functionality.